src/config.rs

```diff
@@ -494,28 +505,52 @@ impl SharedConfigValues {
             }
         }
 
+        cfg_if::cfg_if! {
```

Comment:
i think this might work as well (and be nicer)
```
merge!(general fields)

#[cfg(alsa)]
merge(alsa fields)
```

Reply:
Yep. That's a good one.